README.md

```diff
@@ -707,6 +707,7 @@ API | Description | Auth | HTTPS | CORS |
 | [Jamendo](https://developer.jamendo.com/v3.0/docs) | Music | `OAuth` | Yes | Unknown |
 | [JioSaavn](https://github.com/cyberboysumanjay/JioSaavnAPI) | API to retrive song information, album meta data and many more from JioSaavn | No | Yes | Unknown |
 | [KKBOX](https://developer.kkbox.com) | Get music libraries, playlists, charts, and perform out of KKBOX's platform | `OAuth` | Yes | Unknown |
+| [KSoft.Si](https://api.ksoft.si) | API to get lryics for songs | `apiKey` | Yes | Unknown |
```

Comment:
This project is not just a lyrics API. This project contains other APIs, as you can see in the image below:

ref: https://api.ksoft.si/
![image](https://user-images.githubusercontent.com/50463866/126419498-956f425f-82bf-4c5f-b873-b88cb8e4f186.png)

Therefore, this link (https://api.ksoft.si) does not fit in the music section. The lyrics API can be found at this link: https://docs.ksoft.si/api/lyrics-api

- You can update the link and just add the lyrics API
```suggestion
| [KSoft.Si](https://docs.ksoft.si/api/lyrics-api) | API to get lryics for songs | `apiKey` | Yes | Unknown |
```
- or add all 4 APIs in the sections that best match
   - Image API: https://docs.ksoft.si/api/images-api
   - Bans API: https://docs.ksoft.si/api/bans-api
   - Kumo API: https://docs.ksoft.si/api/kumo-api
   - Lyrics API: https://docs.ksoft.si/api/lyrics-api

- or just add https://api.ksoft.si in a section that best matches (my suggestion is add in development section)

What do you prefer?

Reply:
Made changes